src/App.test.js

```diff
@@ -1,18 +1,63 @@
-import { render, screen, fireEvent } from './test-utils';
+import {
+  fireEvent, render, screen,
+} from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { BrowserRouter as Router } from 'react-router-dom';
+import store from './Redux/configureStore';
 import App from './App';
+import { addCountry } from './Redux/countries/cases';
 
-jest.mock('./redux/countries/api.js');
+test('Should display the header', () => {
+  render(
+    <Provider store={store}>
+      <Router>
+        <App />
+      </Router>
+    </Provider>,
+  );
+  const headerText = screen.getByText(/COVID-19 STATS/i);
+  expect(headerText).toBeInTheDocument();
+});
+
+test('Should go to the details page when a country is clicked', () => {
+  store.dispatch(addCountry({
+    name: 'SPAIN',
+    id: 'spain',
+    total_confirmed: 200000,
+    regions: [],
+  }));
```

Comment:
- [ ] This too
```
  ● Should go to the details page when a country is clicked

    TestingLibraryElementError: Unable to find an element with the text: /SPAIN/i. This could be because the text is broken up by multiple elements. In this case, you can provide a function for your text matcher to make your matcher more flexible.

```